src/core/opamProcess.ml

```diff
@@ -14,12 +14,13 @@ let log ?level fmt =
 
 let default_env =
   let f () = lazy (
-    let env = OpamStd.Env.raw_env () in
     match OpamCoreConfig.(!r.cygbin) with
-    | Some cygbin -> OpamStd.Env.cyg_env ~env ~cygbin ~git_location:OpamCoreConfig.(!r.git_location)
-    | None -> env
+    | Some cygbin ->
+      OpamStd.Env.cyg_env ~cygbin
+        ~git_location:OpamCoreConfig.(!r.git_location) ()
+    | None -> OpamStd.Env.raw_env ()
```

Comment:
I don't mind, but the thing I did like before is that it tried to make it clearer that we are always looking at `OpamStd.Env.raw_env ()`, it's just whether it's had `cygbin`/`git_location` added to it.